### PR TITLE
[TMVA][SOFIE] Add Support for SELU & Sigmoid Operator in Keras Parser

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR adds support for parsing SELU and Sigmoid Activation from Keras models into an RModel object.

## Checklist:

- [x] Tested changes locally
- [x] Tests for Parsing Keras & PyTorch models
- [x]  Add ONNX model with Selu & Sigmoid operator for tests

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default with flags -Dtmva-sofie=On

**Review by lmoneta — APPROVED** [review]
LGTM!
